### ISSUE TITLE
Specify order of columns in project curve table

### ISSUE THREAD

**Issue by kwinkunks** [agile-geoscience/welly#51]
Probably with a flag like `auto_order` which is `True` by default (current behaviour).

**Comment by EvanBianco** [comment]
This doesn't happen for the left most columns, but it does works for the curve columns via the order of the elements in `keys`. 

![image](https://user-images.githubusercontent.com/1699959/32000716-078ecf8e-b96e-11e7-86c9-4f316e7a3ccb.png)

Unless I'm missing a particular use case. I'd say we can close this issue.

**Comment by kwinkunks** [comment]
Right, but the idea is to be able to resort them dynamically according to some criteria. They're arranged according to my `keys` list, but now I want alphabetical, now by the reverse of the count of wells per curve.
